email.js

```diff
@@ -0,0 +1,40 @@
+const nodemailer = require("nodemailer");
+require("dotenv").config();
+
+const transporter = nodemailer.createTransport({
+  host: "smtp.mailgun.org",
+  port: 587,
+  secure: false,
+  auth: {
+    user: process.env.M_USER,
+    pass: process.env.M_PASS,
+  },
+  tls: {
+    rejectUnauthorized: false,
+  },
+});
+
+// висилання ГОЛОВНОГО листа верифікаційного
+const sendVerificationEmail = async (email, verificationToken, next) => {
```

Comment:
```suggestion
const sendVerificationEmail = async (email, verificationToken) => {
```
Tutaj nie uzywaj next, to ma być zwykla funkcja